src/stream/Cargo.toml

```
@@ -28,6 +28,7 @@ enum-as-inner = "0.5"
 fixedbitset = { version = "0.4", features = ["std"] }
 futures = { version = "0.3", default-features = false, features = ["alloc"] }
 futures-async-stream = "0.2"
+governor = { version = "0.5", features = ["std", "dashmap", "jitter"] }
```

You have to add this, otherwise the default features won't be disabled.
```suggestion
governor = { version = "0.5", default-features = false, features = ["std", "dashmap", "jitter"] }
```